### DISCUSSION TITLE
The line of changes not true

### DISCUSSION THREAD

**Discussion by Ahmet122** [mlarhrouch/azure-pipeline-gpt-pr-review#40] [General]
Hi! When I open a Pr, extension works fine but when it commented like ' In line 10 please change 'a' to 'b'....' the 'a' is not in line 10 but in line 30. I think extension can't take change lines correctly. Or just takes changes as line. Could you please help me about how to resolve this issue?

**Comment by akselsoft** [comment]
I think this is likely due to the API doing the extension. Are you using OpenAI or the Azure OpenAI? I have found the Azure OpenAI to be particularly inconsistent with line #s
